test/server/cards/02_SHD/events/Evacuate.spec.ts

```diff
@@ -0,0 +1,104 @@
+describe('Evacuate', function() {
+    integration(function(contextRef) {
+        it('returning all units cards, no leaders', function () {
```

Comment:
this is missing the inner describe(). Even if the setupTest is in the individual cases, we still include the prefix "Evacuate's ability"
I know its missing in a few tests, but I think it's a general best practice.
Also the text in the it clause shoud form a sentence with the text from the describe clause. So I recommend to changing it to something like ```should return all unit cards, except leaders to hand```
AT-ATSuppressor.spec.ts has a great example on this